package.json

```diff
@@ -6,7 +6,7 @@
   "files": [
     "dist/*.js"
   ],
-  "main": "dist/index.js",
+  "main": "dist/metamask.web3.js",
```

Comment:
Is this filename change unrelated to the title/description of this PR (i.e. shipping the library unminified).

Reply:
I updated the readme. This is to standardize how the script is imported in HTML.

Reply:
Also updated the PR description.

Reply:
Ah, that makes more sense :+1: Thanks